source/sdk/lang/types.ooc

```diff
@@ -122,8 +122,10 @@ Closure: cover {
 /** An object storing a value and its class. */
 Cell: class <T> {
     val: __onheap__ T
+	type: Class = null
 
     init: func(=val)
+	init: func ~withtype(=val, =type)
```

Comment:
Oh, the sdk uses spaces, I see.
This is to please the `get` functions in `HashDictionary` - I need to store the classtype `T` in order to check whether the requested type actually exists for a particular key.